public/script.js

```diff
@@ -1613,47 +1607,42 @@ async function Generate(type, automatic_trigger, force_name2) {
                 //if (is_pygmalion && i == chat2.length-1) item='<START>\n'+item;
                 arrMes[arrMes.length] = item;
             } else {
-                console.log('reducing chat.length by 1');
-                i = chat2.length - 1;
+                break;
             }
 
             await delay(1); //For disable slow down (encode gpt-2 need fix)
-            // console.log(i+' '+chat.length);
-
-            count_exm_add = 0;
-
-            if (i === chat2.length - 1) {
-                if (!power_user.pin_examples) {
-                    let mesExmString = '';
-                    for (let iii = 0; iii < mesExamplesArray.length; iii++) {
-                        mesExmString += mesExamplesArray[iii];
-                        const prompt = JSON.stringify(worldInfoString + storyString + mesExmString + chatString + anchorTop + anchorBottom + charPersonality + promptBias + allAnchors);
-                        const tokenCount = getTokenCount(prompt, padding_tokens);
-                        if (tokenCount < this_max_context) {
-                            if (power_user.disable_examples_formatting) {
-                                mesExamplesArray[iii] = mesExamplesArray[iii].replace(/<START>/i, '');
-                            }
+        }
 
-                            if (!is_pygmalion) {
```

Comment:
If `if (power_user.disable_examples_formatting)` above is satisfied, `<START>` will be stripped from the message there, so it makes more sense to put the second `if` behind an `else`, otherwise there might be nothing to replace.